src/compiler/scala/tools/util/JavaxTools.scala

```
@@ -0,0 +1,80 @@
+/* NSC -- new Scala compiler
+ * Copyright 2005-2012 LAMP/EPFL
+ * @author Paul Phillips
```

That's merely in the sense of "the author of our woe." "For there again, against the glass, as if to blight his confession and stay his answer, was the hideous author of our woe--the white face of damnation."  That's the first thing I've ever felt like "re-tweeting."  Or, if you prefer, "the author of our faith."  It amounts to the same.
Re: filemanager, that's an artifact of not wanting to forward calls to fail silently; I want to know what protocol the tool demands. Footnote, javap has its own FileManager (minor extension).  The unsupported API can change, as well.  At least I removed all my printlns.  I'll test the FwdingJFM.